js/packages/components/main/Share.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { FC, ReactNode, useCallback, useEffect, useState } from 'react'
 import { View, Vibration, StatusBar, Text, Share, ScrollView } from 'react-native'
 import { Layout } from '@ui-kitten/components'
-import QRCodeScanner from 'react-native-qrcode-scanner'
```

Comment:
Now that we use expo module, could you use [this method](https://docs.expo.dev/versions/latest/sdk/bar-code-scanner/#barcodescannerscanfromurlasyncurl-barcodetypes) to scan a QRCode from a file?
And add a `gallery` button on the bottom right corner of the scan view that trigger the filepicker?